modules/doc/content/newsletter/2021_01.md

```diff
@@ -0,0 +1,22 @@
+# MOOSE Newsletter (January 2021)
+
+# Improved INL HPC Remote Connectivity
+
+The [hpc_remote.md] instructions have been updated for a better user experience in accessing web-based INL HPC resources. The previous instructions involved using an additional browser with a proxy to access said resources (hpcgitlab, moosebuild, hpcweb, etc). The new instructions utilize a proxy configuration that is much easier to setup for the end-user and do not require an additional browser. For more information, see [hpc_remote.md#socks-proxy].
```

Comment:
```suggestion
The [hpc_remote.md] instructions have been updated for a better user experience in accessing web-based INL HPC resources. The previous instructions involved using an additional browser with a proxy to access said resources (hpcgitlab, moosebuild, hpcweb, etc). The new instructions utilize a proxy configuration that is much easier to setup for the end-user and does not require an additional browser. For more information, see [hpc_remote.md#socks-proxy].
```

Reply:
```
The new instructions ... do not
```

vs

```
The new instructions ... does not
```

?

Reply:
eh, I was thinking "A proxy configuration that does not..."

Reply:
Yeah - I actually had this same struggle when I wrote this. Maybe I should just try to rewrite the sentence

Reply:
It's a clause within the larger sentence, so I think my change is grammatically correct.

Reply:
Maybe something like:
```suggestion
The [hpc_remote.md] instructions have been updated for a better user experience in accessing web-based INL HPC resources. The previous instructions involved using an additional browser with a proxy to access said resources (hpcgitlab, moosebuild, hpcweb, etc). The new proxy configuration file method is much easier to setup for the end-user and does not require an additional browser. For more information, see [hpc_remote.md#socks-proxy].
```

Reply:
You win 👍🏼. I like.